razorpay-sdk/src/Utility.php

```diff
@@ -9,10 +9,20 @@ class Utility
     public function verifyPaymentSignature($attributes)
     {
         $expectedSignature = $attributes['razorpay_signature'];
-        $orderId = $attributes['razorpay_order_id'];
         $paymentId = $attributes['razorpay_payment_id'];
 
-        $payload = $orderId . '|' . $paymentId;
+        if (isset($attributes['razorpay_order_id']) === true)
+        {
+            $orderId = $attributes['razorpay_order_id'];
+
+            $payload = $orderId . '|' . $paymentId;
+        }
+        else
```

Comment:
@mayankamencherla Why are we making the changing in SDK without actually changing the SDK. This will bring inconsistencies.